src/Native/Unix/gen-buildsys-clang.sh

```diff
@@ -7,7 +7,7 @@ if [ $# -lt 4 -o $# -gt 6 ]
 then
   echo "Usage..."
   echo "gen-buildsys-clang.sh <path to top level CMakeLists.txt> <ClangMajorVersion> <ClangMinorVersion> <Architecture> [build flavor] [cmakeargs]"
-  echo "Specify the path to the top level CMake file - <corefx>/src/Native"
+  echo "Specify the path to the top level CMake file - <corefx>/src/Native/Unix"
```

Comment:
It isn't clear to me why we needed to move all these files under a Unix folder. Can you say way?